### PR TITLE
The sudo: tag, Trusty, and Python 3.4 are all End of Life

### PR DESCRIPTION
**Pull request by cclauss**
@ucalyptus

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
-os:
-    - linux
+python:
+    - "2.7"
+    - "3.8"
 
 env:
-    matrix:
-        - TASK=lint
```
Comment (ucalyptus) [inline .travis.yml]: Wasn't aware that this goes here without the matrix.

**Review by ucalyptus — APPROVED** [review]

**Comment by ucalyptus** [comment]
something wrong with the __init__.py

File: .travis.yml (hunk 1 above)
Comment (cclauss) [reply .travis.yml]: (`matrix` is EOL in favour of `jobs`)

**Comment by cclauss** [comment]
`skflow/__init__.py`:24:0: C0305: Trailing newlines (trailing-newlines)

**Comment by cclauss** [comment]
We have to use Python 3.5 instead of 3.8 because of
https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs

File: scripts/travis_install.sh [hunk 2]
```diff
@@ -41,8 +41,8 @@ case "$TASK" in
                     "2.7")
                         TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp27-none-linux_x86_64.whl"
                     ;;
-                    "3.4")
-                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp34-cp34m-linux_x86_64.whl"
+                    "3.5")
+                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl"
```
Comment (ucalyptus) [inline scripts/travis_install.sh:45]: @cclauss This is causing a Missing wheel error at workflow build

Comment (ucalyptus) [reply scripts/travis_install.sh:45]: https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl does not exist

**Review by ucalyptus — CHANGES_REQUESTED** [review]
Find a version that works for 3.5

**Comment by ucalyptus** [comment]
Check #191

**Comment by cclauss** [comment]
Closing in favor of #191

**Pull request closed without merging**